src/snyk/snykCode/error/snykCodeErrorHandler.ts

```diff
@@ -177,9 +179,10 @@ export class SnykCodeErrorHandler extends ErrorHandler implements ISnykCodeError
       this.baseSnykModule.snykCodeOld.clearBundle(); // bundle has expired, trigger complete new analysis
     }
 
-    setTimeout(() => {
-      this.baseSnykModule.runCodeScan().catch(err => this.capture(err, options));
-    }, CONNECTION_ERROR_RETRY_INTERVAL);
+    // Removed this code in order to remove runCodeScan
```

Comment:
Suggestion: I think just removing it instead of commenting it out would be better?